components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -51,53 +51,57 @@
                     [value]="option.key"
                   >{{ option.value }}</option>
                 </select>
               </label>
+              <chef-error
+                *ngIf="ruleForm.get('conditions').controls[i].get('attribute').hasError('required') && ruleForm.get('conditions').controls[i].get('attribute').touched"
+              >Attribute is required</chef-error>
             </chef-form-field>
 
             <chef-form-field class="attribute-field">
               <label>
                 <span class="label">Operator</span>
-                <select ngDefaultControl
-                  formControlName="operator"
-                  (change)="updateConditionValues(condition)"
-                >
-                  <option value='EQUALS'>equals</option>
-                  <option value='MEMBER_OF'>member of</option>
+                <select ngDefaultControl formControlName="operator">
+                  <option *ngFor="let operator of operators"
+                    [value]="operator.key"
+                  >{{ operator.value }}</option>
                 </select>
               </label>
+              <chef-error
+                *ngIf="ruleForm.get('conditions').controls[i].get('operator').hasError('required') && ruleForm.get('conditions').controls[i].get('operator').touched"
+              >Operator is required</chef-error>
             </chef-form-field>
-            
+
             <chef-form-field class="attribute-field">
               <label>
                 <span class="label">Value</span>
                 <chef-input ngDefaultControl
                   formControlName="values"
-                  [value]="getConditionValue(condition.controls.values.value)"
-                  (change)="updateConditionValues(condition)"
+                  [value]="condition.controls.values.value"
                 ></chef-input>
               </label>
-
+              <chef-error
+                *ngIf="(ruleForm.get('conditions').controls[i].get('values').hasError('required') || ruleForm.get('conditions').controls[i].get('values').hasError('pattern')) && ruleForm.get('conditions').controls[i].get('values').dirty"
+              >Value is required</chef-error>
               <small class="help"
                 *ngIf="condition.controls.operator.value === 'EQUALS'">
                   Case sensitive, must match exactly.
               </small>
-
               <small class="help"
                 *ngIf="condition.controls.operator.value === 'MEMBER_OF'">
                   Comma-separated list, at least one must match exactly.
               </small>
-
             </chef-form-field>
+
             <chef-form-field class="attribute-field">
               <label>
                 <span *ngIf="showAndLabel(i)" class="label">AND</span>
               </label>
-              <chef-button tertiary *ngIf="showDelete(i)" (click)="deleteCondition(i)">
+              <chef-button tertiary *ngIf="showDelete()" (click)="deleteCondition(i)">
```

Comment:
No longer row specific; only cares about whether there is more than one row or not. If so, all rows get a button.